tr.html

```diff
@@ -0,0 +1,380 @@
+<!DOCTYPE HTML>
+<!--
+	Dimension by HTML5 UP
+	html5up.net | @ajlkn
+	Free for personal and commercial use under the CCA 3.0 license (html5up.net/license)
+-->
+<html lang="en">
+	<head>
+		<title>Stellar Global</title>
+		<meta charset="utf-8" />
+		<meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
+		<link rel="stylesheet" href="assets/css/main.css" />
+		<noscript><link rel="stylesheet" href="assets/css/noscript.css" /></noscript>
+		<meta property="og:title" content="Stellar Global" />
+		<meta property="og:type" content="website" />
+		<meta property="og:url" content="https://stellar-global.org/" />
+		<meta property="og:image" content="https://raw.githubusercontent.com/Stellar-Global/website/main/images/logo.png" />
+		<meta property="og:description" content="A Perfect Source about Stellar Network." />
+		<meta property="og:locale" content="en_GB" />
```

Comment:
```suggestion
		<meta property="og:locale" content="tr_TR" />
```

lang_Locale change it if i am wrong